receptor_affinity/mesh.py

```diff
@@ -136,16 +137,11 @@ def get_routes(self):
         else:
             return read_and_parse_metrics(routes)
 
-    def validate_routes(self):
+    def validate_routes(self) -> None:
```

Comment:
Even without MyPy integration, having annotations like this helps me communicate intent. On the other hand unused code is buggy code.